### ISSUE TITLE
Need new file checks (iOS)

### ISSUE THREAD

**Issue by dkaushik95** [GantMan/jail-monkey#30]
Hey. I found an article which checks for more file paths for iOS. 

[Jailbreak Detection Methods](https://www.trustwave.com/Resources/SpiderLabs-Blog/Jailbreak-Detection-Methods/)

Should I make a PR to add this?

Thanks.

**Comment by GantMan** [comment]
yes, PR away.  I have no way to test right now though.   So I can merge but not know for sure.

Do you have devices to test?

**Comment by dkaushik95** [comment]
No, I have emulators and devices to work with, but none of them are jailbroken. A part of my security testing team told me that the jailbreak can by bypassed using Xcon. And I was trying to find a way around that. Apparently Nintendo has a solution which we don't know yet.
